### ISSUE TITLE
Research and remove more unused tutorials. 

### ISSUE THREAD

**Issue by CMDWillYang** [opensearch-project/OpenSearch-Dashboards#5446]
pulled from issue #4448. PR #5212 did not remove the below tutorials since some are still accessed via routes and used ([example](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/4448#issuecomment-1758456190)). Please see the original issue and PR for more details and discussions. 

-  src/plugins/home/public/application/components/tutorial.

**Comment by ShatilKhan** [comment]
I can do this one

**Comment by CMDWillYang** [comment]
> I can do this one Please assign me @CMDWillYang

Can you @joshuarrrr? I don't think I can assign issues.

**Comment by ShatilKhan** [comment]
Some updates on the Issue
I found references for all the files inside `src/plugins/home/public/application/components/tutorial`

Now some of these files are interrelated , I assume we can't delete [tutorial.js](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/b3104ce6ca1af405426f59a447effd15ab2a6d4c/src/plugins/home/public/application/components/tutorial/tutorial.js) as it's being referenced in here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d8cbc170d1b8f07b0c2938ce32fe3ff125911030/src/plugins/home/public/application/components/home_app.js#L37

## **Here's the problem**
 [number_parameter](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d8cbc170d1b8f07b0c2938ce32fe3ff125911030/src/plugins/home/public/application/components/tutorial/number_parameter.js#L62-L63) has been referenced in `parameter_form`:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d8cbc170d1b8f07b0c2938ce32fe3ff125911030/src/plugins/home/public/application/components/tutorial/parameter_form.js#L33-L34

Then `parameter_form` is referenced in `instruction_set` but never used:
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d8cbc170d1b8f07b0c2938ce32fe3ff125911030/src/plugins/home/public/application/components/tutorial/instruction_set.js#L34

Then finally we have `instruction_set` referenced in `tutorial` which we can't delete due to it's reference mentioned at the begining:https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d8cbc170d1b8f07b0c2938ce32fe3ff125911030/src/plugins/home/public/application/components/tutorial/tutorial.js#L36

## My Understanding

So basically I can remove `parameter_form` reference from `instruction_set` and delete it since it's never used & then I can also delete `number_parameter` & `string_parameter` since they are not referenced  anywhere else, hopefully that won't give any errors 
@joshuarrrr I'd like to know if this is the right approach
